src/Arcus.Template.WebApi/Startup.cs

```diff
@@ -39,6 +42,20 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddHealthChecks();
+
+//#if DEBUG
+            var openApiInformation = new Info
+            {
+                Title = "Arcus.Template.WebApi",
+                Version = "v1"
+            };
+
+            services.AddSwaggerGen(swaggerGenerationOptions =>
+            {
+                swaggerGenerationOptions.SwaggerDoc("v1", openApiInformation);
+                swaggerGenerationOptions.IncludeXmlComments(Path.Combine(AppContext.BaseDirectory, "Arcus.Template.WebApi.Open-Api.xml"));
```

Comment:
Can we plug in the name of the project that was created based on this template?

Reply:
Yep, all occurences of the project name (Arcus.Template.WebApi) will be replaced by the name you give the created project.